From akpm@linux-foundation.org Tue Oct  2 14:40:55 2007
From: WANG Cong <xiyou.wangcong@gmail.com>
Date: Tue, 02 Oct 2007 14:40:49 -0700
Subject: USB: drivers/usb/misc/sisusbvga/sisusb.c: kill two unused variables
To: greg@kroah.com
Cc: linux-usb-devel@lists.sourceforge.net, akpm@linux-foundation.org, xiyou.wangcong@gmail.com
Message-ID: <200710022140.l92Lenxa023770@imap1.linux-foundation.org>


From: WANG Cong <xiyou.wangcong@gmail.com>

Kill two unused variables in drivers/usb/misc/sisusbvga/sisusb.c.

Signed-off-by: WANG Cong <xiyou.wangcong@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/misc/sisusbvga/sisusb.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/usb/misc/sisusbvga/sisusb.c
+++ b/drivers/usb/misc/sisusbvga/sisusb.c
@@ -3316,8 +3316,6 @@ static struct usb_driver sisusb_driver =
 
 static int __init usb_sisusb_init(void)
 {
-	int retval;
-	struct sisusb_usb_data *sisusb;
 
 #ifdef INCL_SISUSB_CON
 	sisusb_init_concode();
